Makefile

```diff
@@ -12,15 +12,13 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-ifneq (, $(BUILDX_BIN))
-	export BUILDX_CMD = $(BUILDX_BIN)
-else ifneq (, $(shell docker buildx version))
-	export BUILDX_CMD = docker buildx
-else ifneq (, $(shell which buildx))
-	export BUILDX_CMD = $(which buildx)
-else
-	$(error "Buildx is required: https://github.com/docker/buildx#installing")
-endif
+PKG := github.com/docker/compose/v2
+VERSION ?= $(shell git describe --match 'v[0-9]*' --dirty='.m' --always --tags)
+
+CGO_ENABLED ?= 0
+GO111MODULE ?= on
```

Comment:
- If this project cannot be built without go modules, we may not even need this variable (and just hard-code to `on` or `auto` below)
- Same for `CGO_ENABLED` (are there specific builds where it must be enabled? or should it _always_ be off?)

Reply:
Yes might be fine for compose to keep them hardcoded. Don't see any reason to not to looking at current implementation.